labelbox_dev/data_row.py

```diff
@@ -66,14 +67,34 @@ def create_one(dataset_id, data_row: CreateDataRowType):
 
 
 def create(dataset_id,
-           data_rows: List[CreateDataRowType],
+           data_rows: List[CreateDataRowType] = [],
+           data_files: List[str] = [],
```

Comment:
Instead of adding a `data_files` as a new parameter (we should avoid clogging the interface with additional parameters. Every new parameter will need to be understood by users), we should just handle the case when `data_row.row_data` is a local file

```
for dr in data_rows:
  if os.path.exists(dr.row_data);
     # Set row_data as the contents of the file
```